scripts/Tools/case.submit

```diff
@@ -115,7 +116,12 @@ sub main
 	    check_DA_settings();
 	    `./xmlchange --noecho  RUN_WITH_SUBMIT=TRUE`;
 	    $config{RUN_WITH_SUBMIT}='TRUE';
-	    $batchutils->submitJobs();
+            if (defined $opts{'jobid'}) {
```

Comment:
Here's the log:

    BATCH SUBMISSION SO THAT 1ST BATCH HOLDS ON PASSED
    JOBID: -j 1234 or -jobid 1234
    TO THE <casename>.submit in Config folder